_api-reference/index-apis/create-index.md

```diff
@@ -80,7 +80,7 @@ Setting | Description
 index.number_of_shards | The number of primary shards in the index. Default is 1.
 index.number_of_routing_shards | The number of routing shards used to split an index.
 index.shard.check_on_startup | Whether the index's shards should be checked for corruption. Available options are `false` (do not check for corruption), `checksum` (check for physical corruption), and `true` (check for both physical and logical corruption). Default is `false`.
-index.codec | The compression type to use to compress stored data. Available values are `best_compression` and `default`.
+index.codec | The compression type to use to compress stored data. Available values are `best_compression` (optimizes for retrieval speed) and `default` (optimizes for better compression at the expense of speed, leading to smaller data sizes on disk).
```

Comment:
This is backwards :)

`best_compression` -> smaller data size, slower reads
`default` -> optimizes for retrieval speed

Reply:
Thank you, @andrross! Changed.